site/source/docs/getting_started/downloads.rst

```diff
@@ -35,26 +35,59 @@ the first time with:
    download => Download ZIP" on the `emsdk GitHub page
    <https://github.com/emscripten-core/emsdk>`_.
 
-Run the following :ref:`emsdk <emsdk>` commands to get the latest tools from
-GitHub and set them as :term:`active <Active Tool/SDK>`:
+Getting the tools
+^^^^^^^^^^^^
+To get the latest tools from GitHub and set them as :term:`active <Active Tool/SDK>` run the following :ref:`emsdk <emsdk>` commands.
```

Comment:
Drop the `from GitHub` (they are actually common from google storage, not github).